packages/dev/core/src/Engines/WebGL/webGLRenderTargetWrapper.ts

```diff
@@ -43,6 +45,38 @@ export class WebGLRenderTargetWrapper extends RenderTargetWrapper {
      */
     public _currentLOD = 0;
 
+    public override get depthStencilTexture() {
+        return this._depthStencilTexture;
+    }
+
+    public override set depthStencilTexture(texture: Nullable<InternalTexture>) {
+        this._depthStencilTexture = texture;

```

Comment:
Couldnt this leak ? and should we dispose the previous texture ?

Reply:
I don't dispose the existing texture if it exists, because in the frame graph, the texture lifetime is handled by the frame graph texture manager. Maybe I should use a regular function instead of a setter and pass a "disposeExisting" boolean so that the default behavior is to dispose the existing texture?

Reply:
would be safer indeed as I am pretty sure a misuse of this will be hard to find in the future :-)